go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/google/go-cmp v0.5.6
+	github.com/hashicorp/go-version v1.3.0 // indirect
```

Comment:
I'm kind of confused by the `// indirect` here 🤔 Shouldn't it be direct dependency since we import it?

Reply:
I don't know what happened here 🤷 Should be fixed now.